py_mmd_tools/mmd_elements.yaml

```
@@ -324,14 +442,44 @@ data_center:
         separator: ','
         comment: Required
         description: The name of the institution principally responsible for originating this data in the format <long institution name> (<short institution name>). This attribute is recommended by the CF convention.
+        format: "text"
+        requirement_level: "Optional"
     long_name:
     short_name:
   data_center_url:
     acdd:
       publisher_url:
         separator: ','
-        comment: Comma separated list.
+        maxOccurs: '1'
+        minOccurs: '1'
+        comment: Required if not hosted by MET. If multiple are listed, use comma separated list and consistency across fields.
```

```suggestion
        comment: Required. If multiple are listed, use comma separated list and consistency across fields.
```
"Hosted at MET" probably meant adc.met.no here. It is required for data published at data.met.no, at least at the moment. This can probably be relaxed later but it's best if it just says "required" for now, since it comes in the data management handbook.